src/config.h

```diff
@@ -87,8 +87,7 @@ static constexpr auto POTENTIAL_MOVES = NUM_INTERSECTIONS + 1; // including pass
 
 /* Maximum supported batch size for OpenCL.
  */
-static constexpr auto MAX_BATCH = 1;
-static_assert(MAX_BATCH == 1, "MAX_BATCH != 1 not implemented");
+static constexpr auto MAX_BATCH = 8;
```

Comment:
Do we have an actual hard limit? Shouln't we just check cfg_batch_size instead?

OpenCL allocation might need to recheck that the available memory is large enough if cfg_batch_size changes. 

Defaulting it to some larger value is probably going to break too many low RAM cards.

Reply:
Extra memory use from batching is only about 3 MB * batch_size for single precision and half that for half precision. I think that MAX_BATCH can be safely removed and replaced with `cfg_batch_size` in `OpenCL.cpp`.

Reply:
I believe MAX_BATCH is used no where already. I'll remove this.